### PR TITLE
Remove [AssemblyNeutral] attributes

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs [hunk 1]
```diff
@@ -6,9 +6,6 @@
 
 namespace Microsoft.Framework.DependencyInjection.ServiceLookup
```
Comment (pranavkm) [inline src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs]: This type needs to be moved to the Interfaces assembly.

Comment (davidfowl) [inline src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs]: Into the runtime.

Comment (bricelam) [inline src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs]: ...so to the `Microsoft.Framework.Runtime.Interfaces` package in the `Microsoft.Framework.DependencyInjection.ServiceLookup` namespace?

Comment (davidfowl) [inline src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs]: We change the namespace.

Comment (pranavkm) [inline src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs]: This is moving to KRuntime. cc @anurse

Comment (bricelam) [inline src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs]: Removed in 663f81d358ba9b3e7d2b6d69f409b7b2137a3350

**Pull request merged**